5calls/app/src/main/java/org/a5calls/android/a5calls/util/AnalyticsManager.kt

```diff
@@ -1,24 +1,36 @@
 package org.a5calls.android.a5calls.util
 
 import android.content.Context
-import com.wbrawner.plausible.android.Plausible
+import com.onebusaway.plausible.android.Plausible
 import org.a5calls.android.a5calls.BuildConfig
 import org.a5calls.android.a5calls.model.AccountManager
 
+
 class AnalyticsManager {
+    private lateinit var plausible: Plausible
     private val staticProps: Map<String, String> = mapOf(
         "isAndroidApp" to "true",
         "androidAppVersion" to BuildConfig.VERSION_CODE.toString())
+    
+    fun init(context: Context) {
+        plausible = Plausible(context, "5calls.org")
```

Comment:
can you make "5calls.org" a private final or whatever the equivalent is in Kotlin? Something like `PLAUSIBLE_DOMAIN = "5calls.org"`

Reply:
updated, apparently this is the kotlin way

Reply:
Kotlin is weird.